integration-test/Css.ts

```diff
@@ -156,10 +156,10 @@ class CssBuilder<T extends Properties1> {
   get fs0() { return this.add("flexShrink", 0); }
   get fs1() { return this.add("flexShrink", 1); }
   flexShrink(value: Properties["flexShrink"]) { return this.add("flexShrink", value); }
-  get flexRow() { return this.add("flexDirection", "row"); }
-  get flexRowReverse() { return this.add("flexDirection", "row-reverse"); }
-  get flexColumn() { return this.add("flexDirection", "column"); }
-  get flexColumnReverse() { return this.add("flexDirection", "column-reverse"); }
+  get fdRow() { return this.add("flexDirection", "row"); }
+  get fdRowReverse() { return this.add("flexDirection", "row-reverse"); }
+  get fdColumn() { return this.add("flexDirection", "column"); }
+  get fdColumnReverse() { return this.add("flexDirection", "column-reverse"); }
```

Comment:
Could go further and do `fdRow` --> `fdr`, `Flex Direction Row Reverse` --> `fdrr`?